docs/maintain-guides-how-to-create-offline-transaction.md

```diff
@@ -0,0 +1,78 @@
+---
+id: maintain-guides-how-to-create-offline-transaction
+title: Generate an offline transaction
+sidebar_label: Generate an offline transaction
+---
+
+# Generating an offline transaction
+
+There are few different ways to create an offline transaction: by using the [@polkadot@cli-signer](https://github.com/polkadot-js/tools/tree/master/packages/signer-cli), [txwrapper](https://github.com/paritytech/txwrapper), or [Subkey](https://github.com/paritytech/substrate/tree/master/bin/utils/subkey). This guide shows you how to use each one of these for different scenarios. 
+
+## polkadot@cli-signer
+
+This would be the easiest to use if you are looking for a CLI approach and have some basic understanding of public-key cryptography. You can use that to create any transaction and submit it (Extrinsic) to the network directly or you create an offline transaction and then submit it over RPC or [PolkadotJS UI](https://polkadot.js.org/apps/#/toolbox).
+
+
+### Installation
+
+We assume you have installed [NodeJS with npm](https://nodejs.org). Run the following command:
+
+```bash
+npm install -g @polkadot/api-cli
+```
+
+### Creating a transaction
+
+Let's say we want to transfer 0.5 KSM from `DMTHrNcmA8QbqRS4rBq8LXn8ipyczFoNMb1X4cY2WD9tdBX` to `EQ64DQpzBkMcPUqUh7ddvAWmuQFnX3gCPpY8E6F9EsZjHPT`. We need to construct a payload and then use `DMTHrNcmA8QbqRS4rBq8LXn8ipyczFoNMb1X4cY2WD9tdBX` to sign it. 
+
+Executing the following command:
+
+```bash
+yarn run:signer sendOffline --account SENDER_ADDR --ws WEBSOCKET_ENDPOINT RUNTIME_MODULE.FUNCTION RECEIVER_ADDR AMOUNT
```

Comment:
Doesn't using `yarn run:signer` imply you've cloned the repository locally? In the previous step you installed it from npm, shouldn't you be using the command provided in the path?

Reply:
Yes please default to yarn for everything - it works better on Windows and in general.

Reply:
good catch!